pynadjust/surveyconvert/ski-ascii.py

```diff
@@ -14,13 +18,16 @@
 
 import os
 import lxml.etree as ET
+import sys
+
+sys.path.append('C:/path-where-pynadjust-folder-is-located-suggest-/Python/site-packages')
+
```

Comment:
I'd recommend leaving this part out of the version on Github. It's a useful workaround at the moment, but in the not-too-distant future this won't be necessary anymore. If it were to stay in then I'd suggest commenting the sys.path... line so that it doesn't catch people out.